prototypes/final-fixes-shortcuts-autogeneration.lua

```diff
@@ -76,7 +76,8 @@ if autogen_color == "default" or autogen_color == "red" or autogen_color == "gre
 		"wire-cutter-copper",
 		"wire-cutter-green",
 		"wire-cutter-red",
-		"wire-cutter-universal"
+		"wire-cutter-universal",
+		"blueprint-align__item_grid-selection-tool"
```

Comment:
My bad, this is missing an underscore.
```suggestion
		"blueprint-align__item__grid-selection-tool"
```